clients/python/tests/ed25519.private.pem

```diff
@@ -0,0 +1,3 @@
+-----BEGIN PRIVATE KEY-----
+MC4CAQAwBQYDK2VwBCIEIEkA1H+gMmMBURMXBcahTDpffH1nGVPtHF57LnQXJm7h
```

Comment:
Note that I've added two keys inside the tests for `AuthContext`. Since we need tokens in more places now, we should have one keypair somewhere top-level and load them in all tests.